foundry.toml

```diff
@@ -14,6 +14,8 @@ tenderly = "https://rpc.tenderly.co/fork/${TENDERLY_FORK_ID}"
 [profile.default.fmt]
 wrap_comments = true
 
+[profile.default.fuzz]
+max_test_rejects = 131072
```

Comment:
Let's have the same as in the "fast" CI no?

Reply:
I'm ok to do it but it makes the CI longer because there are too many rejection s in `test_withdraw`